Gemfile

```diff
@@ -1,46 +1,31 @@
 source 'https://rubygems.org'
 
+gem 'rails', '~> 5.1.5'
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 5.1.0'
-# Use postgresql as the database for Active Record
-gem "pg", "~> 1.0", :require => false
-# Use Puma as the app server
-gem 'puma', '~> 3.0'
-# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.0'
```

Comment:
Are we even using jbuilder?  Do we expect to?

Reply:
I don't think so, @bdunne ?

Reply:
Just asking for now...I can remove in a separate PR

Reply:
Nope